builtin/plugins/coin/coin.go

```diff
@@ -318,6 +330,9 @@ func (c *Coin) Allowance(
 	ctx contract.StaticContext,
 	req *AllowanceRequest,
 ) (*AllowanceResponse, error) {
+	if ctx.FeatureEnabled(loomchain.CoinVersion1_2Feature, false) && (req.Spender == nil || req.Owner == nil) {
```

Comment:
Like BalanceOf, Allowance does not change state of the chain, so the feature flag check is unnecessary. All methods that use `contract.StaticContext` are quite safe to be changed without a feature flag.